web_src/less/_repository.less

```diff
@@ -1479,8 +1479,12 @@
                 margin-right: 0;
             }
 
+            .svg {
```

Comment:
Why we need this change?

Reply:
To add proper margin on the right of svg